packages/tempus/src/Tempus_IntegratorBasic_impl.hpp

```diff
@@ -413,6 +413,12 @@ void IntegratorBasic<Scalar>::startIntegrator()
     integratorStatus_ = Status::FAILED;
     return;
   }
+  
+  //set the Abs/Rel tolerance
+  auto ws = solutionHistory_->getCurrentState();
+  ws->setTolRel(timeStepControl_->getMaxRelError());
+  ws->setTolAbs(timeStepControl_->getMaxAbsError());
+
```

Comment:
Looks good.  You may want to name the variable cs instead of ws to avoid confusion.